### PR TITLE
request to get parent id for level 2 keywords

### PR DESCRIPTION
**Pull request by Dezedene**
request to get parent id for level 2 keywords

### CODE REVIEW

File: queries/getKeywords.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-const getKeywords = 'SELECT name, importance FROM keyword';
+const getKeywords = 'SELECT k.id, k.name, k.importance, kh.keyword_parent as parent_id,  kh.keyword_children as children_id FROM keyword_hierarchy kh RIGHT JOIN keyword k ON kh.keyword_children = k.id;';
```
Comment (erabin05) [inline queries/getKeywords.js]: Please indent your query

**Review by erabin05 — CHANGES_REQUESTED** [review]
- indentation on query

File: queries/getKeywords.js (hunk 1 above)
Comment (Dezedene) [reply queries/getKeywords.js]: It's done

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**